packages/gui/src/electron/main.tsx

```diff
@@ -639,6 +639,83 @@ if (ensureSingleInstance() && ensureCorrectEnvironment()) {
       mainWindow.setTitle(title);
     }
   });
+
+  // IPC handlers for log file operations
+  ipcMain.handle('getChiaLogContent', async (_event) => {
+    try {
+      const chiaRoot = process.env.CHIA_ROOT || path.join(app.getPath('home'), '.chia', 'mainnet');
+      const logPath = path.join(chiaRoot, 'log', 'debug.log');
```

Comment:
There are users like me who set custom log file location like `log/full_node.log`, `log/wallet.log`, `log/farmer.log` and so on.

So log file location should be loaded and parsed from config file.